packages/google-drive-kit/src/board-server/operations.ts

```diff
@@ -38,6 +38,20 @@ export type GraphInfo = {
 
 let alreadyWarnedAboutMissingPublicApiKey = false;
 
+/** Retries fetch() calls until status is not an internal server error. */
+async function retriableFetch(
+  input: string|Request,
+  init?: RequestInit,
+  numRetries = 3
+): Promise<Response> {
+  let response: Response;
+  do {
+    response = await fetch(input, init);
+    numRetries--;
+  } while (response.status === 500 && numRetries >= 0);
```

Comment:
Probably should check the whole 500 range, something like `status >= 500 && status <= 599`

Reply:
Done